en/micro-integrator/docs/setup/deployment/k8s-pipeline/modifying-pipeline-configurations.md

```diff
@@ -0,0 +1,201 @@
+# Modifying Pipeline Configurations
+
+## The pipeline configurations can be changed by the modification of values.yaml file
```

Comment:
Not suitable for a title. When we render this in the website it appears on the right side navigation and it will display in a lengthy unreadable manner. Please have something shorter.

Reply:
I have added a shorter text